node/full.go

```diff
@@ -266,6 +268,24 @@ func (n *FullNode) OnStart() error {
 	// since p2p pubsub and host are required to create ProofService,
 	// we have to delay the construction until Start and use the help of ProofServiceFactory
 	n.fraudService = n.proofServiceFactory.CreateProofService()
+	if err := n.fraudService.AddVerifier(types.StateFraudProofType, func(fraudProof fraud.Proof) (bool, error) {
```

Comment:
Question for my own understanding.  I assume `AddVerifier` is called on each block?  Is that correct?

Reply:
I think it should only be called once when initializing the node.

Reply:
yes, only one time.